Wahrscheinlichkeit für Kollision für N Teilchen pro Strahl: 

$N^2 \sigma_p /A$ mit $A$ der Strahlquerschnittsfläche

Luminosität:

$ \mathscr{L}=\frac{I_+ I_-}{4\pi e^2 f_{\text{fev}} n_b \sigma_x \sigma_y} = \frac{N_1 N_2 f_{\text{rev}}}{n_b A}=\dot{N_p}\frac{1}{\sigma_p}$

mit:
- $n_b$ der Anzahl der Bunche im Strahl
- $N_1, N_2$ der Teilchen in Bunch 1 und 2
- $\sigma_{x,y}$ der Stahlgröße am Wechselwirkungspunkt
- $N$ der Gesamtteilchenzahl in einem Strahl
- $I=e N f_{\text{rev}}$ dem Strahlstrom
- $A=4\pi\sigma_x\sigma_y$ bei Gaußverteilten Paketen

Ereignisrate:

$\dot{N_p}=(N^2 f_{\text{rev}}/A) \sigma_p=\mathscr{L} \sigma_p$

Rahmenbedingungen:
- mindestens $10^6$ Events pro Jahr bei 75\% Verfügbarkeit der Maschine $\rightarrow$ 0.042279722645019446 Paare pro Sekunde
- Pro Kollision nicht mehr als 5 $t\bar{t}$-Paare 
- Im Strahl gespeicherte Energie $\leq 50\,$MJ $\rightarrow$ Energie im Strahl $=N\cdot \frac{\sqrt{s}}{2}$ oder?!
- Strahlstrom $I\leq1\,\text{A}$
- Umfang 6km

In [2]:
import numpy as np
from scipy import constants 

In [3]:
c=constants.c
e=constants.e

In [4]:
# Länge des Beschleunigers in km
l=6
# Radius in m
r=l*1.E3/(2*np.pi)
# Magnetstärke in Tesla
B=8
# Dipolfüllfaktor
a=0.8
# Ablenkradius im Dipol
r_bending=r*a

In [5]:
# Energy in TeV
energy=r_bending*c*B*1.E-12
E_cm=2*energy

In [6]:
E_cm

3.664393082293972

<img src="./plot.jpg" width="700">

Lese Wirkungsquerschnitt aus Diagramm ab: $\sigma\sim 30\,$pb



In [7]:
sigma_ttbar=30

In [8]:
# Events pro Sekunde
events_per_s=1e6/(0.75*365*24*60*60)
events_per_collision=5

5 Paare pro Kollision,  0.042279722645019446 Paare pro Sekunde

In [9]:
collisions_per_s=5/0.042279722645019446
collisions_per_s

118.26

$\rightarrow$
Kollisionen pro Sekunde: $\dot{N}=118.26\,\text{s}^{-1}$

Luminosität $\mathscr{L}=\dot{N} \cdot \frac{1}{\sigma}$

In [21]:
# Luminosität in m^-2*s^-1
L=collisions_per_s/(sigma_ttbar*1E-40) 

In [23]:
L*1.e-4 # in cm^-2*s^-1

3.9420000000000007e+36

<img src="./luminosity_comparison.jpg" width="300">

In [12]:
# Umlauffrequenz
f_rev=c/l

In [14]:
f_rev/1.e6

49.965409666666666

Umlauffrequenz $f_{rev}\approx 50\,\text{MHz}$

Maximaler Strahlstrom $I=e N f_{\text{rev}}=1\,\text{A}$

In [15]:
# Strahlstrom in A
current=1

In [16]:
# Teilchenzahl pro Strahl
N=current/(e*f_rev)

In [17]:
N

124916599625.6136

$\dot{N_p}=(N^2 f_{\text{rev}}/A) \sigma_p=\mathscr{L} \sigma_p$

In [24]:
# Querschnittsfläche der Bunches im Kollisionspunkt
A=N**2*f_rev/L

In [26]:
A # in m^2

1.9778490363116417e-11

$\rightarrow$ Das ist zu klein, da stimmt was nicht...

In [27]:
stored_energy=energy*1.E12*e*N

In [30]:
stored_energy/1.E6 # in MJ

0.03666929888837269